PET_Clinic/PET_BUSINESS_TRANSACTIONS.md

```diff
@@ -0,0 +1,72 @@
+
+At first, you have to switch to the ![Configuration perspective](../images/compass.png?raw=true) *Configuration perspective*. As in all Eclipse-based application, perspective bar is available on the top-right of your application window.
+
+Open the ![Business Context Manager](../images/briefcase.png?raw=true) **Business Context Manager** and click on the *Create Application* button. In the appearing dialog, you can specify the name of the application and assign a description, for example create the application Pet_Clinic_API which represents the AngularJS frontend (API Gateway):
+
+Property | Value
+--- | ---
+Name | Pet_Clinic_API
+Description | Pet Clinic Microservice application
+
+### Creating Application Definitions
+
+Once a business context has been created, it is opened in the editor. Now, we have to specify the application definition. In order to correlate data to an application **application mappings** are used.
+
+Create a new **application mapping** by clicking on the ![Add](../images/add_obj.gif?raw=true) plus icon in the upper right corner of the view.
+
+We know that each microservice pf the Petclinic application is running on different ports. Therefore, a **HTTP Server Port Matching** should be created which matches all requests on a specific port to an application. The Pet_Clinic_API is running on port 8080. Specify this port and save the configuration. Now, all data matching this application definition is correlated to it. You can check this by navigating to the *Data Explorer View* of the api-gateway agent. 
+
+### Creating Business Transaction Definitions
+
+In the next step, we can group the data of an application and combine them with a certain business context, for example: the business transaction *Owner* contains all data belonging to the process of editing the details of an owner.
+
+#### 1. Creating Business Transaction for Owner
+
+First, switch to the **Business Transaction Definitions** tab (at the bottom of the view) and click on the **Create** button. As before, a dialog appears where we can name the newly created business transaction definition. Enter a name, e.g. *Owner* and click on the *Finish* button.
+
+Like a application definition, a business transaction definition consists of **business transaction mappings** which are used to correlate data to a business transaction.
+
+As before, create an **URI Matcher** which accepts all data where the URI starts with the value ```/api/customer/owners```. Press ```CTRL + S``` to save the changes. And that's it.
+
+#### 2. Creating Business Transaction for Static Resources
+
+Second, we would like to see all requests that are related to static resources. Create a new business tracsaction with the name "Static Resources". As before, create an **URI Matcher** which accepts all data where the URI matches the regex ```.*(js|css|woff|woff2|png)```.  
```

Comment:
Here you can say that static resources are all requests that have URI ending in .js / .css / etc, thus we will use URI regex matcher..